### PR TITLE
Update README Structure 

### PR DESCRIPTION
**Pull request by Nash0x7E2**
Update the README to use a tweaked version of our open-source template.

### CODE REVIEW

**Comment by Nash0x7E2** [comment]
Would love to get some feedback on this one. Before merging, we should update the tutorial section with links to each SDK, forgot that part. 

cc: @jeroenleenarts for your awareness

**Comment by jeroenleenarts** [comment]
We should get some nice image at the top as well. Everything else looks good.

For the tutorials, are you referring to the base Chat ones?

If so, here are all the links we need, ready to be dropped in. Note that the JS one if actually a link to the JS client docs.

```
* [React](https://getstream.io/chat/react-chat/tutorial/)
* [React](https://getstream.io/chat/react-native-chat/tutorial/)
* [Angular](https://getstream.io/chat/angular/tutorial/)
* [Jetpack Compose](https://getstream.io/tutorials/android-chat/)
* [SwiftUI](https://getstream.io/tutorials/ios-chat/)
* [Flutter](https://getstream.io/chat/flutter/tutorial/)
* [Javascript/Bring your own](https://getstream.io/chat/docs/javascript/)
```

File: README.md [hunk 1]
```diff
@@ -42,12 +42,66 @@ StreamingMessageView(
...
+## 🛥 What is Stream?
+
+Stream allows developers to rapidly deploy scalable feeds, chat messaging and video with an industry leading 99.999% uptime SLA guarantee.
+
+Stream provides UI components and state handling that make it easy to build real-time chat and video calling for your app. Stream runs and maintains a global network of edge servers around the world, ensuring optimal latency and reliability regardless of where your users are located.
+
+## 📕 Tutorials
+
+To learn more about integrating AI and chatbots into your application, we recommend checking out the full list of tutorials across all of our supported frontend SDKs and providers. Stream's Chat SDK is natively supported across:
+* React
```
Comment (jeroenleenarts) [inline README.md]: If it is chat tutorial links you are looking for:

```
* [React](https://getstream.io/chat/react-chat/tutorial/)
* [React](https://getstream.io/chat/react-native-chat/tutorial/)
* [Angular](https://getstream.io/chat/angular/tutorial/)
* [Jetpack Compose](https://getstream.io/tutorials/android-chat/)
* [SwiftUI](https://getstream.io/tutorials/ios-chat/)
* [Flutter](https://getstream.io/chat/flutter/tutorial/)
* [Javascript/Bring your own](https://getstream.io/chat/docs/javascript/)
```

**Review by jeroenleenarts — CHANGES_REQUESTED** [review]
The links need updating, made a suggestion there. Might need an image too at the top.

File: README.md (hunk 1 above)
Comment (martinmitrevski) [reply README.md]: I added those, the second one was missing Native (for React Native).

**Review by martinmitrevski — APPROVED** [review]

**Comment by martinmitrevski** [comment]
@jeroenleenarts updated the links & I'm going to merge this, cover image would be done later after Nash and Josh figure this out.

**Review by jeroenleenarts — COMMENTED** [review]
LGTM

**Pull request merged**